src/Components/ServiceScene/LogListControls.tsx

```diff
@@ -3,6 +3,7 @@ import React, { useCallback } from 'react';
 import { css } from '@emotion/css';
 
 import { GrafanaTheme2, LogsSortOrder } from '@grafana/data';
+import { t } from '@grafana/i18n';
```

Comment:
Ah we doing this now for all our text?

Reply:
We should as a best effort for new strings, maybe we want to spike to get everything else translated and/or add eslint rules to require translation for copy that is modified/changed over time